rel-jdk8-overrides.yaml

```diff
@@ -28,8 +28,17 @@ packages:
   manager: yum
   content_sets_file: content_sets_rhel7.yml
 
+
 osbs:
-      repository:
-            name: containers/jboss-eap-74-openjdk8-builder
-            branch: jb-eap-7.4-rhel-7
+  configuration:
```

Comment:
our JDK8 images are not impacted by the multiarch build, why do we need to update that config?

Reply:
I dropped the container.yaml file that includes the only: x86_64, so this is making the remaining build configs consistent.